nexus/types/src/external_api/params.rs

```diff
@@ -1242,17 +1245,7 @@ pub struct AddressLotCreate {
     /// The kind of address lot to create.
     pub kind: AddressLotKind,
     /// The blocks to add along with the new address lot.
-    pub blocks: Vec<AddressLotBlockCreate>,
-}
-
-/// Parameters for creating an address lot block. Fist and last addresses are
-/// inclusive.
-#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
-pub struct AddressLotBlockCreate {
-    /// The first address in the lot (inclusive).
-    pub first_address: IpAddr,
-    /// The last address in the lot (inclusive).
-    pub last_address: IpAddr,
+    pub blocks: Vec<IpRange>,
```

Comment:
better but still problematic, right? I think we need to change this API to mimic what we do for ip-pools